x-pack/plugins/security_solution/public/sourcerer/experimental/readme.md

```diff
@@ -0,0 +1,18 @@
+# Experimental Sourcerer Replacement
+
+## Introduction
+
+This directory is a home for Discovery Components based re-implementation of the Sourcerer.
+
+Currently, it can be enabled and used only by setting the localStorage value, like this:
+
+```
+window.localStorage.setItem('EXPERIMENTAL_SOURCERER_ENABLED', true)
+```
+
+The reason for having this feature toggle like this is we want to be able to inspect both implementations side by side,
+using the same Kibana instance deployed locally (for now).
```

Comment:
Would like to understand more here? If we want to inspect this locally only, Why do we need to merge this code?

Reply:
We need to allow people to test it immediately, without the need to switch the feature flag and redeploy - and side by side to identify potential mismatches in data presented in the app.

Reply:
you do set it per the entire deployment right?

Reply:
yes? But what do you mean by deployment? 😆

Reply:
How do you do feature flag that is disabled in one browser window and enabled in the other one, both operating on the same kibana instance?